lib/generators/blacklight/blacklight_access_controls.rb

```diff
@@ -0,0 +1,19 @@
+Blacklight::AccessControls.configure do |config|
+  # This specifies the solr field names of permissions-related fields.
+  # The default fields used are shown below, if you index your permissions to other fields update the configuration below.
+  # If you change these, you must also update the permissions request handler in your solrconfig.xml to return those values
+  #
+  # config.permissions.discover.group       = Solrizer.solr_name("discover_access_group", :symbol)
+  # config.permissions.discover.individual  = Solrizer.solr_name("discover_access_person", :symbol)
+  # config.permissions.read.group           = Solrizer.solr_name("read_access_group", :symbol)
+  # config.permissions.read.individual      = Solrizer.solr_name("read_access_person", :symbol)
+  # config.permissions.edit.group           = Solrizer.solr_name("edit_access_group", :symbol)
+  # config.permissions.edit.individual      = Solrizer.solr_name("edit_access_person", :symbol)
```

Comment:
The Config class for blacklight-access_controls is much simpler than the Hydra Config class.  It doesn't have all the wrappers and hash accessors, so you can just set values directly, like this:

```
config.discover_group_field = "discover_access_group_ssim"
```
The method names are here:
https://github.com/projectblacklight/blacklight-access_controls/blob/master/lib/blacklight/access_controls/config.rb#L18-L21